.travis.yml

```diff
@@ -87,22 +90,34 @@ jobs:
         - npm install
         - npm run build
 
+      before_deploy:
+        - ~/.local/bin/aws s3 rm s3://$AWS_S3_BUCKET_PROD/$DEPLOY_PATH --recursive --region $AWS_S3_REGION
+
       deploy:
-        provider: script
-        script: ~/.local/bin/aws s3 sync dist s3://$AWS_S3_BUCKET_PROD/kuzzle-v1 --delete
+        provider: s3
+        access_key_id: $AWS_ACCESS_KEY_ID
+        secret_access_key: $AWS_SECRET_ACCESS_KEY
+        bucket: $AWS_S3_BUCKET_PROD
         skip_cleanup: true
+        upload-dir: $DEPLOY_PATH
+        local-dir: dist
+        region: $AWS_S3_REGION
+        on:
+          all_branches: true
 
       after_deploy:
-        - aws cloudfront create-invalidation --distribution-id $AWS_CLOUDFRONT_DISTRIBUTION_ID_PROD --paths "/kuzzle-v1/*"
+        - ~/.local/bin/aws cloudfront create-invalidation --distribution-id $AWS_CLOUDFRONT_DISTRIBUTION_ID_PROD --paths "/$DEPLOY_PATH/*"
 
     # ---------------------------------------
-    # next.console.kuzzle.io
+    # console.kuzzle.io/kuzzle-v2
```

Comment:
Like the other PR, why don't we just keep v2 deployments on the v2 branches?

Reply:
I'm putting the PR back in draft !